backend/rates/rates.go

```diff
@@ -122,6 +123,38 @@ func (updater *RateUpdater) Last() map[string]map[string]float64 {
 	return updater.last
 }
 
+// LastForPair returns the conversion rate for the given (coin, fiat) pair. Returns an error if the
+// rates have not been fetched yet.
+func (updater *RateUpdater) LastForPair(coinCode string, fiat string) (float64, error) {
+	// TODO: use coin.Code
+	last := updater.Last()
+	if last == nil {
+		return 0, errp.New("rates not available yet")
+	}
+	key := map[string]string{
```

Comment:
I would move this to the top, next to `coins` package var.

Reply:
Done.